projects/aca-content/src/lib/store/effects/node.effects.spec.ts

```diff
@@ -564,4 +566,40 @@ describe('NodeEffects', () => {
       expect(store.dispatch).toHaveBeenCalledWith(new NavigateUrlAction('personal-files/details/node-id?location=test-page'));
     });
   });
+
+  describe('folderInformation$', () => {
+    it('should call folder information dialog', () => {
+      const node: any = { entry: { isFile: true } };
```

Comment:
No need to use any. You can use 'as NodeEntry' and correct type will be set

Reply:
Replaced any with NodeEntry